runbot_travis2docker/models/runbot_repo.py

```diff
@@ -40,3 +40,4 @@ def weblate_validation(self):
             json = response.json()
             if 'projects' not in json:
                 raise ValidationError(_('Response json bad formated'))
+        raise Warning(_('Connection with weblate successful'))
```

Comment:
@JulioSerna Thanks for you help
@moylop260 
In this file https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/ir/ir_mail_server.py#L199
It is used `#L199 raise UserError(_("Connection Test Succeeded! Everything seems properly set up!"))`
I think which in this case is the same what `ir_mail_server#test_smtp_connection`